src/metabase/channel/render/card.clj

```diff
@@ -145,6 +149,26 @@
           (log/error e "Pulse card render error")
           (body/render :render-error nil nil nil nil nil))))))
 
+(def branding-logo-base64-src
+  "Returns a Base64 data URI of a PNG logo from assets."
+  (->
```

Comment:
we probably want to wrap this in a `(delay ...)` and then when we use it below dereference it with `@branding-logo-base64-src`

Reply:
+1 same with `branding-content` so it doesn't take space in memory until it does.

also if it's static I'd say render it with hiccup so it's a string and takes up way less memory

Reply:
https://github.com/metabase/metabase/pull/57434/commits/8894abad69610e3161fee0b34f3e54257804250c